tests/Storage/Compression/Algorithms/ZopfliTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Utopia\Tests\Storage\Compression\Algorithms;
+
+use Utopia\Storage\Compression\Algorithms\Zopfli;
+use PHPUnit\Framework\TestCase;
+
+class ZopfliTest extends TestCase
+{
+    protected Zopfli $object;
+
+    public function setUp(): void
+    {
+        $this->object = new Zopfli();
+    }
+
+    public function tearDown(): void
+    {
+    }
+
+    public function testName()
+    {
+        $this->assertEquals($this->object->getName(), 'zopfli');
```

Comment:
The first parameter for `assertEquals()` should be the expected value and the 2nd should be the actual.

Reply:
okay I got it!